src/kapla/projects/pyproject.py

```diff
@@ -340,12 +341,13 @@ async def poetry_publish(
 
     @classmethod
     def find_current(
-        cls: Type[PyProjectT], start: Union[None, str, Path] = None
+        cls: Type[PyProjectT], start: Union[None, str, Path] = None, venv_path: str = None
```

Comment:
We should remove this argument as that's now the method which is getting itself the variable from environment variables.

Reply:
Yes